ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py

```diff
@@ -190,6 +190,38 @@ def test_sparse_categorical_crossentropy(
     )
 
 
+# log_cosh
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        shared_dtype=True,

```

Comment:
You could let the `shared_dtype` be `False` by default, just to check that the type promotion is working correctly.